superset/assets/javascripts/explore/components/controls/BoundsControl.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Col, Row, FormGroup, FormControl } from 'react-bootstrap';
+import ControlHeader from '../ControlHeader';
+
+const propTypes = {
+  name: PropTypes.string.isRequired,
+  label: PropTypes.string,
+  description: PropTypes.string,
+  onChange: PropTypes.func,
+  value: PropTypes.array,
+};
+
+const defaultProps = {
+  label: null,
+  description: null,
+  onChange: () => {},
+  value: [null, null],
```

Comment:
I thought about this and wanted to minimize the `formData` payload to a minimum, at least until we use POST instead of GET throughout.